### PR TITLE
Accordion and Modal spec updates

### PR DESCRIPTION
**Pull request by coliff**
## Description

- Correct font size for accordion header
- Correct font size for modal header and body text
- Added `role="button"` and `aria-label="Close"` to Modal Close button (accessibility fix)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?

Locally with Edge v103 on Windows 10

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (including CHANGELOG updates)
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-modal/modus-modal.tsx [hunk 1]
```diff
@@ -88,8 +88,8 @@ export class ModusModal {
         <div class="content">
           <div class="header">
             {this.headerText}
-            <div class="icon-close" onClick={() => this.close()} >
-              <IconClose size="20"/>
+            <div class="icon-close" role="button" aria-label="Close" onClick={() => this.close()}>
```
Comment (ryan-henness-trimble) [inline stencil-workspace/src/components/modus-modal/modus-modal.tsx:91]: Did the tests need to be updated to include this?

File: stencil-workspace/src/components/modus-text-input/readme.md [hunk 2]
```diff
@@ -15,7 +15,7 @@
 | `errorText`         | `error-text`          | (optional) The input's error state text.                      | `string`                                                                    | `undefined` |
 | `helperText`        | `helper-text`         | (optional) The input's helper text displayed below the input. | `string`                                                                    | `undefined` |
 | `includeSearchIcon` | `include-search-icon` | (optional) Whether the search icon is included.               | `boolean`                                                                   | `undefined` |
-| `inputmode`         | `inputmode`           | (optional) The input's inputmode.                             | `"decimal" \| "email" \| "numeric" \| "search" \| "tel" \| "text" \| "url"` | `'text'`    |
+| `inputmode`         | `inputmode`           | (optional) The input's inputmode.                             | `"decimal" \| "email" \| "numeric" \| "search" \| "tel" \| "text" \| "url"` | `undefined` |
```
Comment (ryan-henness-trimble) [inline stencil-workspace/src/components/modus-text-input/readme.md:18]: Was this manually updated?

File: stencil-workspace/src/components/modus-modal/modus-modal.scss [hunk 3]
```diff
@@ -3,13 +3,15 @@
...
   font-family: $primary-font;
+  font-size: 0.875rem;
   height: 100%;
   justify-content: center;
   left: 0;
   overflow: auto;
   position: fixed;
   top: 0;
   width: 100%;
+  z-index: 1055 !important;
```
[the remark this answers is not in the document]
Comment (coliff) [reply stencil-workspace/src/components/modus-modal/modus-modal.scss:14]: I think I can remove that now. It was to deal with an issue with the code block appearing above… but it was the code block z-index that caused the issue which I fixed in that PR that was merged yesterday.

File: stencil-workspace/src/components/modus-text-input/readme.md (hunk 2 above)
Comment (coliff) [reply stencil-workspace/src/components/modus-text-input/readme.md:18]: no it was updated automatically, it should have been included in a previous PR. The above change is correct. inputmode should not be defined by default.

File: stencil-workspace/src/components/modus-modal/modus-modal.tsx (hunk 1 above)
Comment (coliff) [reply stencil-workspace/src/components/modus-modal/modus-modal.tsx:91]: I'm not sure but tests are passing as they are...

**Review by ryan-henness-trimble — APPROVED** [review]

**Pull request merged**